### ISSUE TITLE
[pytket-cirq] Cirq 0.11 breaks pytket-cirq

### ISSUE THREAD

**Issue by cqc-alec** [CQCL/pytket-extensions#79]
pytket-cirq has the requirement `cirq ~= 0.10`, which accepts 0.11 (released yesterday). This breaks some of our code:
```
ModuleNotFoundError: No module named 'cirq.devices'
```
We should (a) make pytket-cirq compatible with cirq 0.11, and (b) change the requirement to `cirq ~= 0.11.0` so that this doesn't happen next time.

**Comment by cqc-alec** [comment]
In fact I can't get cirq 0.11 to work at all, it seems to be broken internally:

```python
import cirq_google
```

errors out with:

```
ImportError: cannot import name 'devices' from 'cirq' (unknown location)
```

Will pin pytket-cirq to 0.10 for now...

**Comment by cqc-alec** [comment]
Released pytket-cirq 0.10.1 which forces use of cirq 0.10. But we should try and support 0.11 if possible.

**Comment by cqc-alec** [comment]
> In fact I can't get cirq 0.11 to work at all, it seems to be broken internally:

Just tried this again in a brand-new python environment and it works.
